Symfony/CS/Fixer/PSR2/EofEndingFixer.php

```diff
@@ -23,15 +25,25 @@ class EofEndingFixer extends AbstractFixer
      */
     public function fix(\SplFileInfo $file, $content)
     {
-        // [Structure] A file must always end with a linefeed character
+        $tokens = Tokens::fromCode($content);
 
-        $content = rtrim($content);
+        $count = $tokens->count();
+        if (0 === $count) {
+            return;
```

Comment:
good point, I've changed this https://github.com/SpacePossum/PHP-CS-Fixer/commit/78029db016ad131c431905ed5e8ddb6f1d02db85